app/views/templates/_form.html.erb

```diff
@@ -38,8 +38,9 @@
 
 
       <% if pxe_with_building_hosts?(@template) -%>
+        <% warning_text = (_("Warning: The template is associated to at least one host in build mode. To apply the change, disable and enable build mode on hosts to update the live templates or choose to %s their configuration from 'Select Action' menu") % link_to(_('recreate'), hosts_path(:host_ids => building_hosts(@template).map(&:id)))).html_safe %>
```

Comment:
This link loads the hosts index, but unless you happen to have the cookie already then the select action dropdown isn't shown, and the hosts aren't selected.